addons/web_dashboard/controllers/main.py

```diff
@@ -0,0 +1,68 @@
+from openerp import tools
+from openerp.addons.web import http
+from openerp.addons.web.http import request
```

Comment:
Import the native object, not the "already imported one"
``
from openerp import http, tools
from openerp.http import request
``